completions_v2.ts

```diff
@@ -0,0 +1,198 @@
+// Copyright 2022 the Deno authors. All rights reserved. MIT license.
```

Comment:
Why is this file at the root of the tree?

Reply:
well, it isnt part of pages, static, utils, nor tests, so it has no place to belong, + this is being imported directly by main.ts